t/file-writelines.neon

```diff
@@ -0,0 +1,31 @@
+IMPORT file
+
+VAR lines: Array<String> := ["Hello", "Neon", "", "World!"]
+
+TRY
+    file.writelines("linetest.txt", lines)
+EXCEPTION file.FileOpenError
+    print("write file open error")
+EXCEPTION file.FileWriteError
+    print("file write error")
+END TRY
+
+VAR data: Array<String>
+
+TRY
+    data := file.lines("linetest.txt")
+EXCEPTION file.FileOpenError
+    print("read file open error")
+END TRY
+
+print("\(data.size()) lines read.")
+%= 4 lines read.
+
+FOREACH s OF data DO
+    print("\(s)")
```

Comment:
Since `s` is itself a string, interpolation is redundant here and `print(s)` is simpler.